### PR TITLE
fix one cause of 499s

### PR DESCRIPTION
**Pull request by szuecs**
fix context issue causing some 499s with probability around 0.3% under high load

Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: cluster/manifests/skipper/daemonset.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: kube-system
   labels:
     application: skipper-ingress
-    version: v0.10.112
+    version: v0.10.121-1-g5f1806d
```
Comment (aermakov-zalando) [inline cluster/manifests/skipper/daemonset.yaml:8]: Should we cherry-pick it to `.112`, considering that `.121` wasn't tested in the clusters, or is it fine? What other changes are there?

Comment (szuecs) [reply cluster/manifests/skipper/daemonset.yaml:8]: Only filters added which we do not enable by default and one fix for kubernetes in GCP. 
I think it's fine.

**Comment by szuecs** [comment]
:+1:

**Comment by linki** [comment]
:+1:

**Pull request merged**